app/views/subscription/index.phtml

```diff
@@ -47,7 +47,7 @@
 
 						$error_class = '';
 						$error_title = '';
-						if ($feed->inError()) {
+						if ($feed->inError() && !$feed->mute()) {
```

Comment:
Subscription management is quite different from just the UI like in `index/global.html`. Which isn't to say that it definitely should show them, but the considerations are different.

Reply:
Yes, the consideration is different, but I find it more consistent and also pleasing with this change. Don't you think?

Reply:
I agree about aesthetically pleasing, but not about consistent. If "muting" and "unmuting" seemingly toggles the error status I find that inconsistent. As such the current display makes sense to me:
![image](https://github.com/FreshRSS/FreshRSS/assets/202757/d27dcb24-eee3-4cef-8318-3500e67b494f)

Reply:
It was to be consistent with the main UI. The error status of muted feeds is not reliable as it is never updated, so it stays to the last status. I do not really mind reverting this page though

Reply:
To me it makes sense that in management you'd want more info than in the main UI. In this case it's an implicit clue about why you muted it. For the EFIC example above I'd otherwise wonder why I muted that and I'd unmute it right back.

Put another way, there's muting because it's definitely dead and there's muting because you don't want updates at this time but you don't want to delete it (yet).

Reply:
Ok, reverted

Reply:
Good now?

Reply:
Should be good, yes.